plugins/junit-xml-plugin/src/main/java/io/qameta/allure/junitxml/JunitXmlPlugin.java

```diff
@@ -127,11 +130,21 @@ private void parseTestSuite(final XmlElement testSuiteElement, final Path parsed
                         resultsDirectory, parsedFile, context, visitor));
     }
 
+    private LocalDateTime getZuluTime(final String timestamp) {
+        DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'").withZone(ZoneOffset.UTC);
+        return LocalDateTime.parse(timestamp, dtf);
+    }
+
     private Long getUnix(final String timestamp) {
         if (isNull(timestamp)) {
             return null;
         }
-        final LocalDateTime parsed = LocalDateTime.parse(timestamp);
+        final LocalDateTime parsed;
+        if (timestamp.matches(ZULU_TIME_FORMAT)) {
```

Comment:
maybe we should use try/catch instead of matching by regexp? Like iterate over supported formats and trying to parse until found one that matches

Reply:
See no problem with try...catch block this makes logic more bullet proof and more readable.  My rational behind regex was an assumption that matcher should be slightly easier for JVM rather than exception.

I'll update the code with try catch and fallback to zulu format. Do you think it will make sense to check regex before parse in catch block?

Reply:
The solution I have in mind is to have a list of different data formats and iterate over formats trying to parse. If no format found log an exception and use null value. This way it would be easy to add support for different formats and even and an extension point to support custom user formats (that would be useful in case of custom framework that logs in extended junit xml format)

Reply:
> and even and an extension point to support custom user formats (that would be useful in case of custom framework that logs in extended junit xml format)

PS no need to implement this part right now